experimental/packages/opentelemetry-sdk-metrics/src/export/PeriodicExportingMetricReader.ts

```diff
@@ -50,21 +43,21 @@ export type PeriodicExportingMetricReaderOptions = {
   exportTimeoutMillis?: number;
 };
 
-const DEFAULT_AGGREGATION_SELECTOR: AggregationSelector = Aggregation.Default;
-
 /**
  * {@link MetricReader} which collects metrics based on a user-configurable time interval, and passes the metrics to
- * the configured {@link MetricExporter}
+ * the configured {@link PushMetricExporter}
  */
 export class PeriodicExportingMetricReader extends MetricReader {
   private _interval?: ReturnType<typeof setInterval>;
   private _exporter: PushMetricExporter;
   private readonly _exportInterval: number;
   private readonly _exportTimeout: number;
-  private readonly _aggregationSelector: AggregationSelector;
 
   constructor(options: PeriodicExportingMetricReaderOptions) {
-    super();
+    super({
+      aggregationSelector: options.exporter.selectAggregation,
```

Comment:
`options.exporter.selectAggregation` should either have their receiver to be bound to `options.exporter`, or maybe we should make the selector an interface, with methods like `selectAggregation` and `selectAggregationTemporality`, so that we can safely pass the selector around.

```js
interface AggregationSelector {
  selectAggregation(instrumentType): Aggregation;
  selectTemporality(instrumentType): AggregationTemporality;
}
```

Reply:
Good catch, thanks! :slightly_smiling_face: 
I changed it to bind to the exporter in [2453614](https://github.com/open-telemetry/opentelemetry-js/pull/3225/commits/2453614f029999d07c9064e92359fab20290dec3). :slightly_smiling_face: